css/build.css

```diff
@@ -142,6 +142,10 @@
 }
 
 /* For native devices */
-.native .fa-times-thin {
-  padding-top: 4px;
+.native.android .fa-times-thin {
+  padding-top: 5px;
+}
+
+.native.iphonex .fa-times-thin {
```

Comment:
Can you confirm that you're only adding this override to iPhone X (or iPhone X-like devices), and not to all iOS devices? I believe there's a `.native.ios` selector that you can use if you want to target all iOS devices.

Reply:
@tonytlwu No, I was targeting all iOS devices. Sorry I didn't notice this class. I will make new PR to change it right away.